angular/projects/spark-angular/src/lib/components/sprk-promo/sprk-promo.stories.ts

```diff
@@ -86,21 +118,49 @@ flag.story = {
 export const withImage = () => ({
   moduleMetadata: modules,
   template: `
-    <sprk-promo
-      title="Title"
-      subtitle="Subtitle"
-      media="img"
-      imgAlt="placeholder"
-      imgSrc="https://spark-assets.netlify.app/house.jpg"
-      cta="button"
-      ctaText="Learn More"
-      buttonHref="#"
-      idString="promo-3"
-      hasBorder="true"
-    >
-      Lorem ipsum dolor. Sit amet pede. Tempus donec et. Suspendisse id
-      inventore integer eum non enim diam habitant. Maecenas nunc per lacus
-      neque egestas. Diam quod curabitur.
+    <sprk-promo idString="image-promo" hasBorder="true" additionalClasses="sprk-o-Stack--split@s">
+      <a href="#nogo" sprkStackItem class="sprk-o-Stack__item--half@s">
+       <img
+         class="sprk-c-Promo__image"
+         alt="A house and front yard."
```

Comment:
Before the alt was just set to "placeholder". I updated it to be more descriptive of the image.